packages/explat-client/src/internal/test/timing.ts

```diff
@@ -25,43 +25,45 @@ const delayedValue = < T >( value, delayMilliseconds ): Promise< T > =>
 
 describe( 'timeoutPromise', () => {
 	it( 'should resolve promises below the timeout', async () => {
-		expect( Timing.timeoutPromise( new Promise( ( res ) => res( 123 ) ), 1 ) ).resolves.toBe( 123 );
-		expect( Timing.timeoutPromise( delayedValue( 123, 1 ), 2 ) ).resolves.toBe( 123 );
+		await expect( Timing.timeoutPromise( new Promise( ( res ) => res( 123 ) ), 1 ) ).resolves.toBe(
+			123
+		);
+		await expect( Timing.timeoutPromise( delayedValue( 123, 1 ), 2 ) ).resolves.toBe( 123 );
 	} );
 	it( 'should throw if promise gets timed-out', async () => {
-		expect( Timing.timeoutPromise( delayedValue( null, 2 ), 1 ) ).rejects.toThrowError();
-		expect( Timing.timeoutPromise( delayedValue( null, 3 ), 2 ) ).rejects.toThrowError();
+		await expect( Timing.timeoutPromise( delayedValue( null, 2 ), 1 ) ).rejects.toThrowError();
```

Comment:
This was another thing I fixed in the next few PRs but didn't have fixed here, this can cause unfortunately cause other unrelated tests to fail. 

@scinos I wish there was a lint rule for this. There is one for the alternate convention of `expect(await promise)`: [jest/no-restriced-matches](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-restricted-matchers.md)

Reply:
This rule may work https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/valid-expect.md ?